test/test_dna.py

```diff
@@ -263,6 +263,34 @@ def testMatchWithAmbiguityInSecond(self):
             compareDNAReads(Read('id1', 'ACGTTC'),
                             Read('id2', 'ACGTTS')))
 
+    def testNonMatchingAmbiguityInFirst(self):
+        """
+        Two sequences that match exactly, apart from one ambiguity in the
```

Comment:
I would say "apart from one non-matching ambiguity".

Reply:
Or "incompatible" (as with your next test that uses "compatible").

Reply:
done